server/neptune/template/loader.go

```diff
@@ -0,0 +1,73 @@
+package template
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"os"
+
+	_ "embed" // embedding files
+
+	"github.com/Masterminds/sprig/v3"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+	"github.com/runatlantis/atlantis/server/events/models"
+)
+
+type Key string
+
+// input for the template to be loaded
+// add fields here as necessary
+type Input struct {
+}
```

Comment:
I would define this in the file you're using it if possible since this type will be different for whatever it's calling.  And also just call it, LegacyApplyCommentInput or something specific so its clear.

Reply:
Or call it empty so its clear that we shouldn't add to it.